doc/changelog/11-standard-library/18668-depr_NDefOps_NIso.rst

```diff
@@ -0,0 +1,5 @@
+- **Deprecated:**
+  The library files ``Coq.Numbers.Natural.Abstract.NIso``
+  and ``Coq.Numbers.Natural.Abstract.NDefOps`` are deprecated.
```

Comment:
There should be no period here (it's two line below). Could you also take the opportunity to fix all other entries in this `11-standard-library` directory.